# ExecuteOberservingPath.srv
geometry_msgs/Point input_point
---
bool success
